src/site/content/en/blog/coop-coep/index.md

```diff
@@ -0,0 +1,306 @@
+same-origin policy.)
+
+{% Aside 'caution' %}
+These powerful features and the prevention of `document.domain` modification 
+are not yet enabled in Chrome as of version 83. We'll update this post as they 
+become available.
+{% endAside %}
+
+To opt in to a cross-origin isolated state, you need to send the following two 
+HTTP headers on the main document:
+
+```http
+Cross-Origin-Embedder-Policy: require-corp
+Cross-Origin-Opener-Policy: same-origin
+```
+
+These headers instruct the browser to block loading of resources or iframes 
+which haven't opted into being loaded by cross-origin documents, and prevent 
+cross-origin windows from directly interacting with your document. This also 
+means those resources being loaded cross-origin require opt-ins.
+
+You can determine whether a web page is in a cross-origin isolated state by 
+examining `self.crossOriginIsolated`. (This works on Firefox but has yet to be 
+implemented in Chrome).
+
+This article shows how to use these new headers. In a follow-up article I will
+provide more background and context.
+
+{% Aside %}
+This article is aimed at those who would like to get their websites ready for 
+using `SharedArrayBuffer`, WebAssembly Threads, `performance.measureMemory()`
+or the JS Self-Profiling API in a more robust manner across browser 
+platforms.
+{% endAside %}
+
+{% Aside 'key-term' %}
+This article uses many similar-sounding terminologies. To make things 
+clearer, let's define them first:
+
+* [COEP: Cross Origin Embedder 
+  Policy](https://wicg.github.io/cross-origin-embedder-policy/)
+* [COOP: Cross Origin Opener 
+  Policy](https://github.com/whatwg/html/pull/5334/files)
+* [CORP: Cross Origin Resource 
+  Policy](https://developer.mozilla.org/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP))
+* [CORS: Cross Origin Resource 
+  Sharing](https://developer.mozilla.org/docs/Web/HTTP/CORS)
+* [CORB: Cross Origin Read 
+  Blocking](https://www.chromium.org/Home/chromium-security/corb-for-developers)
+{% endAside %}
+
+## Deploy COOP and COEP to make your website cross-origin isolated
+### Integrate COOP and COEP
+#### 1. Set the `Cross-Origin-Opener-Policy: same-origin` header on the top-level document
+
+By enabling COOP on a top-level document, windows 
+with the same origin, and windows opened from the document, will have a separate 
+browsing context group unless they are in the same origin with the same COOP 
+setting. Thus, isolation is enforced for opened windows.
+
+A browsing context group is a group of tabs, windows or iframes which share the
+same context. For example, if a website (`https://a.example`) opens a popup
+window (`https://b.example`), the opener window and the popup window share the
+same browsing context and they have access to each other via DOM APIs such as
+`window.opener`.
+
+![Browsing Context Group](browsing-context-group.png)
+
+As of Chrome 83, DevTools support is not yet available for COOP. However, you
```

Comment:
"... dedicated DevTools support is not ..." is probably a better and true wording, as window.opener===null for example can be checked via DevTools' console as a debugging help.